x-pack/plugins/security_solution/public/management/cypress/support/agent_actions.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// / <reference types="cypress" />
+import type { ExecaReturnValue } from 'execa';
+import execa from 'execa';
+import { agentRouteService } from '@kbn/fleet-plugin/common/services';
+import { API_VERSIONS } from '@kbn/fleet-plugin/common/constants';
+import type { GetOneAgentResponse } from '@kbn/fleet-plugin/common/types';
+
+import { VAGRANT_CWD } from '../../../../scripts/endpoint/common/endpoint_host_services';
+import { createRuntimeServices } from '../../../../scripts/endpoint/common/stack_services';
+
+export const agentActions = (
+  on: Cypress.PluginEvents,
+  config: Cypress.PluginConfigOptions
+): void => {
+  const stackServicesPromise = createRuntimeServices({
+    kibanaUrl: config.env.KIBANA_URL,
+    elasticsearchUrl: config.env.ELASTICSEARCH_URL,
+    fleetServerUrl: config.env.FLEET_SERVER_URL,
+    username: config.env.KIBANA_USERNAME,
+    password: config.env.KIBANA_PASSWORD,
+    esUsername: config.env.ELASTICSEARCH_USERNAME,
+    esPassword: config.env.ELASTICSEARCH_PASSWORD,
+    asSuperuser: true,
+  });
+
+  on('task', {
+    uninstallAgentFromHost: async ({
```

Comment:
can you add definitions for these task under `x-pack/plugins/security_solution/public/management/cypress/cypress.d.ts`